test/test_other.py

```diff
@@ -14285,22 +14285,30 @@ def test_pthread_kill(self):
   @node_pthreads
   @parameterized({
     '': ([], 1),
+    'growable_arraybuffers': (['-sGROWABLE_ARRAYBUFFERS'], 1),
     'proxy': (['-sPROXY_TO_PTHREAD', '-sEXIT_RUNTIME'], 2),
   })
   def test_pthread_growth_mainthread(self, cflags, pthread_pool_size):
+    if '-sGROWABLE_ARRAYBUFFERS' in cflags:
+      self.node_args.append('--experimental-wasm-rab-integration')
+      self.require_node_canary()
```

Comment:
Maybe worth adding a code size test for this, which would verify we are shipping a lot less code? Or some other kind of check that we aren't instrumenting the code in the slow way.

Reply:
Hmm .. good idea.